### PR TITLE
cmd: fix import/export from file, allow stdin/stdout to be used

### PR DESCRIPTION
**Pull request by guggero**
Fixes https://github.com/lightninglabs/taro/issues/141 and two general bugs in the CLI.

### CODE REVIEW

File: cmd/tarocli/proofs.go [hunk 1]
```diff
@@ -34,8 +36,9 @@ var verifyProofCommand = cli.Command{
 	Description: "verify a taro proof",
 	Flags: []cli.Flag{
 		cli.StringFlag{
-			Name:  proofPathName,
-			Usage: "the file path to the .taro file",
+			Name: proofPathName,
+			Usage: "the path to the proof file on disk; use the " +
+				"dash character (-) to read from stdin instead",
```
Comment (bhandras) [inline cmd/tarocli/proofs.go:41]: nit: instead, would it be cleaner to use a bool flag?

**Review by bhandras — APPROVED** [review]
LGTM :tada:

File: cmd/tarocli/proofs.go [hunk 2]
```diff
@@ -51,7 +52,8 @@ func verifyProof(ctx *cli.Context) error {
 		return nil
 	}
 
-	rawFile, err := os.ReadFile(proofPathName)
+	filePath := lncfg.CleanAndExpandPath(ctx.String(proofPathName))
```
Comment (Roasbeef) [inline cmd/tarocli/proofs.go:58]: 👍

File: cmd/tarocli/proofs.go [hunk 3]
```diff
@@ -168,3 +184,41 @@ func importProof(ctx *cli.Context) error {
 	printRespJSON(resp)
 	return nil
 }
+
+// readFile attempts to read a file from disk. If the passed fileName is equal
+// to the dash character, then this function reads from stdin instead.
+func readFile(fileName string) ([]byte, error) {
```
Comment (Roasbeef) [inline cmd/tarocli/proofs.go:190]: 👍

**Review by Roasbeef — APPROVED** [review]
LGTM 📍

**Pull request merged**